### PR TITLE
chore(sql): add config option to disable parallel SQL filter execution

### PR DESCRIPTION
**Pull request by puzpuzpuz**
JIT compiled filters are kept available only when parallel SQL filters are enabled.

Also updates AWS marketplace config artifact with the contents of the default `server.conf`. The reason for that it that they were totally out of sync.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 9 / 9 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|5|5|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 9 / 9 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|5|5|100.00%|

**Review by bluestreak01 — DISMISSED** [review]

File: pkg/ami/marketplace/assets/server.conf [hunk 1]
```diff
@@ -398,23 +493,34 @@ line.tcp.auth.db.path=conf/auth.txt
...
+
+# SO_RCVBUF value, -1 = OS default
+#pg.net.connection.rcvbuf=-1
+
+# SO_SNDBUF value, -1 = OS default
+#pg.net.connection.sndbuf=-1
+
 #pg.character.store.capacity=4096
 #pg.character.store.pool.capacity=64
 #pg.connection.pool.capacity=64
```
Comment (bziobrowski) [inline pkg/ami/marketplace/assets/server.conf:519]: Doesn't 1-per-boot.sh rely on pg.password=PG_PASSWORD_REPLACE  ?

File: core/src/main/resources/io/questdb/site/conf/server.conf [hunk 2]
```diff
@@ -394,6 +394,9 @@ query.timeout.sec=60
 
 ################ Parallel SQL execution ################
```
Comment (bziobrowski) [inline core/src/main/resources/io/questdb/site/conf/server.conf:396]: I think we should mention here that jit won't work unless  parallel filter is enabled .

File: pkg/ami/marketplace/assets/server.conf (hunk 1 above)
Comment (puzpuzpuz) [reply pkg/ami/marketplace/assets/server.conf:520]: Good spot. Restored all original prop values in 4fe2eb84fb9bbc868fad01e3404b921ec07d6e7f

File: core/src/main/resources/io/questdb/site/conf/server.conf (hunk 2 above)
Comment (puzpuzpuz) [reply core/src/main/resources/io/questdb/site/conf/server.conf:396]: Expanded the comment in 5fb86a0e03b358a46a449c8f81e584e40a45af2e

**Review by bziobrowski — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 9 / 9 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCodeGenerator.java|5|5|100.00%|

**Pull request merged**